cmd/artifactPrepareVersion.go

```diff
@@ -497,10 +497,6 @@ func propagateVersion(config *artifactPrepareVersionOptions, utils artifactPrepa
 	}
 
 	for i, targetTool := range config.AdditionalTargetTools {
-		if targetTool == config.BuildTool {
-			// ignore configured build tool
-			continue
-		}
```

Comment:
Is this change related to npm or something else?

Reply:
yes this the change needed if users want to proporgate the changes from parent package.json to sub packages, or else sub packages will continue to have the same version as before